examples/dummy_plugin/dummy_plugin/templates/dummy_plugin/dummymodel.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'base.html' %}
+{% load buttons %}
+{% load custom_links %}
+{% load helpers %}
+{% load plugins %}
+{% load static %}
+{% load tz %}
```

Comment:
Are `static` and `tz` actually needed here? I'm not generally loading them in my plugin "detail" templates.

Reply:
Just more copypasta. I can remove it for brevity.